engine/main.py

```diff
@@ -1077,25 +1081,45 @@ def _create_post(group, subject, tag_info, message_text, user, sender_addr, msg_
 
         tag_objs = Tag.objects.filter(tagthread__thread=thread)
         tags = list(tag_objs.values('name', 'color'))
-        group_members = MemberGroup.objects.filter(group=group)
         recipients = []
-        for m in group_members:
-            # print "create post", user.email
-            print "donotsend", m.member.email
-            dm = DoNotSendList.objects.filter(group=group, user=user, donotsend_user=m.member)
-            blockingMode = getattr(m, 'tag_blocking_mode')
-            if dm.exists():
-               continue 
-            elif not m.no_emails and m.member.email != sender_addr:
-                muted_tags = MuteTag.objects.filter(tag__in=tag_objs, group=group, user=m.member)
-                if blockingMode:
-                    if not muted_tags.exists():
-                        recipients.append(m.member.email)
+        is_moderated = True # TODO: check if mailing list group is moderated
+
+        if is_moderated:
+            moderators = list(group_members.filter(moderator=True))
+            
+            for mod in moderators:
+                recipients.append(mod.member.email)
+            
+            try:
+                mail = MailResponse(From = sender_addr,Subject = "Post Request Pending")
+                mail.Html = 'You are receiving this message because there is a pending post request in the Murmur group %s. View post <a href="/thread?group_name={{ %s }}&tid={{%s}}">here</a>. <br />' %(group,group,p.msg_id)
```

Comment:
Can you make these lines that formatting an email as a function in utils.py? Something like this: https://github.com/haystack/murmur/blob/d10808a92f7a39e1df34c326ef61661f89e16161/smtp_handler/utils.py#L164-L192

Reply:
Also, can you add Not only Html but also the plain version of email?